<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Workspace</class>
 <widget class="QWidget" name="Workspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>276</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QListWidget" name="listOpenedFiles">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of Grouped items:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_num_groupped">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="new_array">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/gimp-pro/images/gtk-add.png</normaloff>:/icons/resources/gimp-pro/images/gtk-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_array">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/gimp-pro/images/gtk-remove.png</normaloff>:/icons/resources/gimp-pro/images/gtk-remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="display_array">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/gimp-pro/images/gtk-home.png</normaloff>:/icons/resources/gimp-pro/images/gtk-home.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="duplicate_array">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/resources/other_icons/copy-item.png</normaloff>:/resources/other_icons/copy-item.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="psh_move_up">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/gimp-pro/images/gtk-go-up.png</normaloff>:/icons/resources/gimp-pro/images/gtk-go-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="psh_move_down">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/resources/gimp-pro/images/gimp-merge-down.png</normaloff>:/icons/resources/gimp-pro/images/gimp-merge-down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
